sup3r/postprocessing/writers/nc.py

```diff
@@ -0,0 +1,132 @@
+"""Output handling
+
+TODO: Remove redundant code re. Cachers
+"""
+
+import json
+import logging
+from datetime import datetime as dt
+
+import numpy as np
+import xarray as xr
+
+from sup3r.preprocessing.names import Dimension
+
+from .base import OutputHandler
+
+logger = logging.getLogger(__name__)
+
+
+class OutputHandlerNC(OutputHandler):
+    """OutputHandler subclass for NETCDF files"""
+
+    # pylint: disable=W0613
+    @classmethod
+    def _get_xr_dset(
+        cls,
+        data,
+        features,
+        lat_lon,
+        times,
+        meta_data=None,
+        max_workers=None,  # noqa: ARG003
+        gids=None,
+    ):
+        """Convert data to xarray Dataset() object.
+
+        Parameters
+        ----------
+        data : ndarray
+            (spatial_1, spatial_2, temporal, features)
+            High resolution forward pass output
+        features : list
+            List of feature names corresponding to the last dimension of data
+        lat_lon : ndarray
+            Array of high res lat/lon for output data.
+            (spatial_1, spatial_2, 2)
+            Last dimension has ordering (lat, lon)
+        times : pd.Datetimeindex
+            List of times for high res output data
+        meta_data : dict | None
+            Dictionary of meta data from model
+        max_workers: None | int
+            Has no effect. Compliance with parent signature.
+        gids : list
+            List of coordinate indices used to label each lat lon pair and to
+            help with spatial chunk data collection
+        """
+        coords = {
+            Dimension.TIME: times,
+            Dimension.SOUTH_NORTH: lat_lon[:, 0, 0],
```

Comment:
A question. It is always guarantee that latitude repeat along the first dimension and longitude repeat along the second dimension, i.e. `lat_lon[:,0,0] == lat_lon[:,1,0]`

Reply:
No and this is definitely not the case for WTK or NSRDB data, thats why I keep the 2d array for lat / lon coords. I actually just added south_north / west_east for easier plotting with hvplot, but then I found `.quadmesh` so I can remove this.